ReactAndroid/src/main/java/com/facebook/react/uimanager/UIImplementation.java

```diff
@@ -44,6 +47,14 @@
  */
 public class UIImplementation {
 
+  static {
+    try {
+      mReactBaseTextShadowNodeClass = Class.forName("com.facebook.react.views.text.ReactBaseTextShadowNode");
```

Comment:
Without using reflection, to break circular dependency of buck, there will be up to 795 lines changed. And Warnings:
![image](https://user-images.githubusercontent.com/2918923/50509169-e34cc000-0abf-11e9-9dd8-c9e6ffe5645d.png)